lib/vmpooler.rb

```diff
@@ -18,4 +18,47 @@ module Vmpooler
       require File.expand_path(File.join(File.dirname(__FILE__), 'vmpooler', lib))
     end
   end
+
+  def self.config
+    # Load the configuration file
+    config_file = File.expand_path('vmpooler.yaml')
+    $config = YAML.load_file(config_file)
```

Comment:
Is there a reason we still need to use global variables with all of this in `self.config` now?